README.md

```diff
@@ -40,7 +40,7 @@ Go to the administrative panel of your hosting in order to learn how to set up t
 
 The following cron Tasks must be configured:
 
-`*/5 * * * * php /var/www/html/modules/reverb/cron.php?code=product > /var/log/cron.log`
+`*/5 * * * * php /var/www/html/modules/reverb/cron.php?code=products > /var/log/cron.log`
```

Comment:
any other places where we need to change this? https://github.com/reverbdotcom/prestashop/search?p=2&q=code%3Dproduct&type=&utf8=%E2%9C%93

Reply:
I fix in FAQ and Spanish Readme too